main.go

```diff
@@ -46,6 +46,9 @@ func run() {
 	toiletPaperEntities := []entities.ToiletPaper{}
 	virusEntities := []entities.Virus{}
 
+	soundEffectsBuffer := resources.GetSoundEffectsBuffer()
+	resources.PlayMenuMusic()
```

Comment:
Dont change to this but would a singleton instance of a sound manager make more sense for something like this?